modules/nw-sriov-hwol-configuring-machine-config-pool.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-hardware-offloading.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-machine-config-pool_{context}"]
+= Configuring a machine config pool for hardware offloading
+
+To enable hardware offloading, you must first create a dedicated machine config pool and configure it to work with the SR-IOV Network Operator.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Create a machine config pool for machines you want to use hardware offloading on.
+
+.. Create a file, such as `mcp-offloading.yaml`, with content like the following example:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: mcp-offloading <1>
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - {key: machineconfiguration.openshift.io/role, operator: In, values: [worker,mcp-offloading <1>]}
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/<offload-name>: ""
+----
+<1> The name of your machine config pool for hardware offloading.
+
+.. Apply the configuration for the machine config pool:
++
+[source,terminal]
+----
+$ oc create -f mcp-offloading.yaml
+----
+
+. Add nodes to the machine config pool. Label each node by running the following command:
++
+[source,terminal]
+----
+$ oc label node <node_name> node-role.kubernetes.io/<offload_name>=""
+----
+
+. Optional: To verify that the new pool is created, run the following command:
+[source,terminal]
+----
+$ oc get nodes
+----
++
+.Example output
```

Comment:
The label renders as `+ .Example output` in the published output.

https://deploy-preview-41762--osdocs.netlify.app/openshift-enterprise/latest/networking/hardware_networks/configuring-hardware-offloading.html#about-hardware-offloading_configuring-hardware-offloading

Reply:
Fixed, thanks!